openpype/hosts/maya/api/pipeline.py

```diff
@@ -426,6 +468,49 @@ def on_before_save():
     return lib.validate_fps()
 
 
+def after_file_open():
+    """Check if there is a user opening the file"""
+    log.info("Running callback on checking the lock file...")
+
+    # add the lock file when opening the file
+    filepath = current_file()
+
+    if not is_workfile_locked(filepath):
+        create_workfile_lock(filepath)
+
+    else:
+        username = get_username(filepath)
+        reminder = cmds.window(title="Reminder", width=400, height=30)
+        cmds.columnLayout(adjustableColumn=True)
+        cmds.separator()
+        cmds.columnLayout(adjustableColumn=True)
+        comment = " %s is working the same workfile!" % username
+        cmds.text(comment, align='center')
+        cmds.text(vis=False)
+        cmds.rowColumnLayout(numberOfColumns=3,
+                             columnWidth=[(1, 300), (2, 100)],
+                             columnSpacing=[(2, 10)])
+        cmds.separator(vis=False)
+        quit_command = "cmds.quit(force=True);cmds.deleteUI('%s')" % reminder
+        cmds.button(label='Ok', command=quit_command)
+        cmds.showWindow(reminder)
+
+
+def on_before_close():
+    """Delete the lock file after user quitting the Maya Scene"""
+    log.info("Closing Maya...")
+    # delete the lock file
+    filepath = current_file()
+    remove_workfile_lock(filepath)
+
+
+def before_file_open():
+    """check lock file when the file changed"""
+    log.info("check lock file when file changed...")
```

Comment:
I feel like this logging could be clearer.

```suggestion
    log.info("Removing lock on current file before scene open...")
```

Or more generic which could be in the lock workfile api instead:
```suggestion
    log.debug("Removing lock on current file...")
```

I actually wonder whether maybe this should be debug logging in the `openpype.pipeline.lock_workfile` logic instead?